### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by alejx**
1. Handle error code when adding unsupported subtitles and audios.
2. Enable dragging subtitles in VideoView.
3. Change the original behavior in PlaylistDragDestView when dragging. It will not add non-video files into playlist now.

### CODE REVIEW

File: iina/VideoView.swift [hunk 1]
```diff
@@ -400,5 +405,16 @@ class VideoView: NSOpenGLView {
...
   }
-
+  
+  override func draggingEntered(_ sender: NSDraggingInfo) -> NSDragOperation {
+    return .copy
+  }
+  
+  override func performDragOperation(_ sender: NSDraggingInfo) -> Bool {
+    let board = sender.draggingPasteboard()
+    let fileURL: NSURL? = NSURL(from: board)
```
Comment (lhc70000) [inline iina/VideoView.swift]: This solution only supports single file.

Please refer to `PlaylistDragDestView.swift`.

File: iina/VideoView.swift [hunk 2]
```diff
@@ -400,5 +405,16 @@ class VideoView: NSOpenGLView {
...
+    playerCore.loadExternalSubFile(fileURL as! URL)
```
Comment (lhc70000) [inline iina/VideoView.swift]: Maybe need to check whether it's a subtitle file, if not, load the dragged file (as video).

File: iina/PlayerCore.swift [hunk 3]
```diff
@@ -278,14 +278,28 @@ class PlayerCore: NSObject {
...
+  func failedLoadExternalAudioFile(_ returnCode: Int32) {
+    if returnCode >= 0 {
+      return
+    }
+    Utility.showAlert(message: "Unsupported external audio file.")
+  }
+  
   func loadExternalAudioFile(_ url: URL) {
-    mpvController.command(.audioAdd, args: [url.path])
+    mpvController.command(.audioAdd, args: [url.path], checkError: false, returnValueCallback: failedLoadExternalAudioFile)
```
Comment (lhc70000) [inline iina/PlayerCore.swift]: I prefer using a block here. What do you think?

File: iina/PlayerCore.swift [hunk 4]
```diff
@@ -278,14 +278,28 @@ class PlayerCore: NSObject {
...
+  func failedLoadExternalSubFile(_ returnCode: Int32) {
+    if returnCode >= 0 {
+      return
+    }
+    Utility.showAlert(message: "Unsupported external subtitle.")
+  }
+  
   func loadExternalSubFile(_ url: URL) {
-    mpvController.command(.subAdd, args: [url.path])
+    mpvController.command(.subAdd, args: [url.path], checkError: false, returnValueCallback: failedLoadExternalSubFile)
```
Comment (lhc70000) [inline iina/PlayerCore.swift]: same as above

File: iina/PlayerCore.swift (hunk 3 above)
Comment (alejx) [inline iina/PlayerCore.swift]: Sure. Revise later.

File: iina/VideoView.swift (hunk 2 above)
Comment (alejx) [inline iina/VideoView.swift]: Do you have a list of the support format of subtitles, audio tracks and videos? I think it helps to handle.

**Pull request merged**